pkg/workflow/controller/controllers/configmap.go

```diff
@@ -63,6 +55,7 @@ func NewConfigMapController(client clientset.Interface, namespace string, cm str
 		clientSet:    client,
 		informer:     informer,
 		queue:        queue,
+		drCollection: drCollection,
```

Comment:
@angao  I cached the deleted resource object, get and use it in worker processing, is there any problem with this approach? please help me review this. Thanks.

Reply:
Do you read the code of k8s controller? What is its practice? @zhujian7

Reply:
Although it may be not a good practice, but it is a simple way to implement the deletion logic.

Reply:
The finalizer is a better implementation approach, but the change will be huge even uncontrollable in this PR. We can consider changing it to the finalizer way in the future.